app/views/categories/_filter_labels.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="filters-remove">
+  <div class="filters-remove__heading">
+    <div class="govuk-body filters-remove__heading-title govuk-!-margin-bottom-0">
+      Active filters
```

Comment:
please localise this (i.e. set `categories.filter_labels.active_filters` in `en.yml` and use `t(".active_filters") here`)